### PR TITLE
Update "Using environment builds locally"

### PR DESCRIPTION
**Pull request by pjlee11**
~Resolves #NUMBER~

**Overall change:** _Make the read me accurate_

**Code changes:**

~- _A bullet point list of key code changes that have been made._~
~- _When describing code changes, try to communicate **how** and **why** you implemented something a specific way, not just **what** has changed._~

---

~- [ ] I have assigned myself to this PR and the corresponding issues~
~- [ ] Tests added for new features~
- [ ] Test engineer approval
- [x] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,10 +56,14 @@ To avoid indexing by search engines during our early development, there is a `no
 This is mainly used for debugging `latest` using the TEST and LIVE environment bundles. Ensure that the bundles exist in the static asset location for the correct environment before starting to debug.
 
 To run TEST bundles on localhost:
-`rm -rf build && npm run build:test && npm run start`
+- In `envConfig/test.env` change the value of a to `LOG_DIR='/var/log/simorgh'`
```
Comment (ChrisBAshton) [inline README.md]: "change the value of a to"

?

**Review by ChrisBAshton — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (pjlee11) [reply README.md]: 😂  🤦‍♂️

Comment (pjlee11) [reply README.md]:
```suggestion
- In `envConfig/test.env` change the value of `LOG_DIR='/var/log/simorgh'` to `LOG_DIR='log'`
```

File: README.md [hunk 2]
```diff
@@ -56,10 +56,14 @@ To avoid indexing by search engines during our early development, there is a `no
...
+- Then run `rm -rf build && npm run build:test && npm run start`
+- Visit a test article: http://localhost:7080/news/articles/cyddjz5058wo
 
 To run LIVE bundles on localhost:
-`rm -rf build && npm run build:live && npm run start`
+- In `envConfig/live.env` change the value of a to `LOG_DIR='/var/log/simorgh'`
```
Comment (pjlee11) [inline README.md]:
```suggestion
- In `envConfig/live.env` change the value of `LOG_DIR='/var/log/simorgh'` to `LOG_DIR='log'`
```

**Review by dr3 — APPROVED** [review]

**Review by ChrisBAshton — APPROVED** [review]

**Comment by jamesbrumpton** [comment]
👍

**Pull request merged**